sphinx_doc/user_guide.rst

```diff
@@ -24,26 +24,28 @@ Getting started
   * Use ``python manage.py createsuperuser``.
   * Follow the instructions shown on your terminal, then browse and login to the admin page of genomics-workspace. Usually, the admin page should be at ``http://127.0.0.1:8000/admin/``.
   * If you already have an admin account, use ``python manage.py runserver`` and then browse and login to genomics-workspace.
+  
+* All opeations on the command line will be done as the i5k user which is a admin user. You can switch to the i5k user with ``sudo su -l i5k``. You need to be in the apollo group to be able to run this command.
 
 * Create these directories if you don’t have them
 
-  * media/blast/db
-  * media/hmmer/db
+  * /usr/local/i5k/media/blast/db/
+  * /usr/local/i5k/media/hmmer/db
 
 * Create sequence sequence types in blast/sequence-type. We recommend creating these three:
 
   * Peptide/Protein
   * Nucleotide/Genome Assembly
   * Nucleotide/Transcript
 
-* Copy all fasta files to be formatted for blast to media/blast/db
+* Copy all fasta files to be formatted for blast to /usr/local/i5k/media/blast/db/
 
-* Copy protein fasta files to be formatted for hmmer to media/hmmer/db
+* Copy protein fasta files to be formatted for hmmer to /usr/local/i5k/media/hmmer/db
```

Comment:
We can leave this here for now, but hmmer will also read from the media/blast/db directory... perhaps we should talk to @g8tor about best practices for this

Reply:
Good point